### ISSUE TITLE
Error 'sinon not defined' when running tests in --production mode

### ISSUE THREAD

**Issue by mcbain** [practicalmeteor/meteor-sinon#11]
Testing with --production flag ensures that the source wich gets deploymed afterwards was tested.
Sometimes - like in this issue - things do not work after getting minified.

**Comment by ryneeverett** [comment]
@mcbain As of #9, the package is `debugOnly`.

**Comment by mcbain** [comment]
It's the debugOnly flag

**Comment by ryneeverett** [comment]
@mcbain Yeah, it's poorly named. It prevents the package from being included in production bundles.

**Comment by mcbain** [comment]
I just read about it - seems that --production testing is dead :(

**Comment by ryneeverett** [comment]
@mcbain Please post a comment here if you find a solution. Your use case _ought_ to be supported.

**Comment by rbabayoff** [comment]
It can be easily fixed by supporting a DISABLE_DEBUG_ONLY=1 env var and if it is set, not set debugOnly in package.js. PRs welcome.

**Comment by CaptainGones** [comment]
ok ty
